static/src/javascripts/boot.js

```diff
@@ -132,6 +132,59 @@ const go = () => {
 
             // ------------------------------------------------------
 
+            // ------------------------------------------------------
+            // Sending Consent Data to Ophan as its own record
+
+            /*
+
+                Date: Dec 2023
+                Author: Anna Voelker
+
+                We reproduce here the same code that we had developed for DCR:
+                https://github.com/guardian/dotcom-rendering/pull/9546
+            */
+
+                if (!consentState) return;
+
+                const consentData = () => {
+                    if (consentState.tcfv2) {
+                        const consentUUID = getCookie('consentUUID') || '';
+                        const consent = consentState.tcfv2?.tcString;
+                        const consentJurisdiction = "TCF";
+                        return {
+                            consentJurisdiction,
+                            consentUUID,
+                            consent
+                        };
+                    }
+                    if (consentState.ccpa) {
+                        const consentUUID = getCookie('ccpaUUID') || '';
+                        const consent = consentState.ccpa?.doNotSell ? 'false' : 'true';
+                        const consentJurisdiction = "CCPA";
+                        return {
+                            consentJurisdiction,
+                            consentUUID,
+                            consent
+                        };
+                    }
+                    if (consentState.aus) {
+                        const consentUUID = getCookie('ccpaUUID') || '';
+                        const consent = consentState.aus.personalisedAdvertising? 'true': 'false';
+                        const consentJurisdiction = "AUS";
+                        return {
+                            consentJurisdiction,
+                            consentUUID,
+                            consent
+                        };
+                    }
+                    return {};
+                };
+
+
+                ophan.record({consentData});
```

Comment:
I’m puzzled as to the actual shape of the sent object. [In the DCR PR](https://github.com/guardian/dotcom-rendering/pull/9546/files#diff-4bab34d6e24320942820bd0975c532e967d54c3f313d8d719694dc81dcbf7a7aR128) we call `consentDetails()` as the argument of the record function. I am assuming this is what we want here, too?

```suggestion
                ophan.record(consentData());
```